src/utils.js

```diff
@@ -15,9 +15,12 @@ export function getServiceName(obj) {
 	} else {
 		throw new Error(`getServiceName must be called with a string url or with a SystemJS load object that has an address`);
 	}
-	const splits = address.split('/');
 
-	return splits[splits.length - 1];
+	let urlParts = getRegex().exec(address);
```

Comment:
These changes fix `getServiceName` to resolve the service name properly when there is a path in it.